conf/carbon.conf.example

```diff
@@ -311,6 +311,11 @@ MAX_DATAPOINTS_PER_MESSAGE = 500
 # thousands of TCP connections reduce the throughput of the service.
 #MAX_RECEIVER_CONNECTIONS = inf
 
+# Specify the user to drop privileges to
+# If this is blank carbon-relay runs as the user that invokes it
+# This user must have write access to the local data directory
```

Comment:
This comment is inaccurate and unnecessary. Only carbon-cache requires write permissions to the data directory since it's the only daemon type that actually writes to Whisper. Would you mind removing this line and the one below for aggregator?